src/index.js

```diff
@@ -852,6 +860,56 @@ class Offline {
     clearTimeout(timeout);
   }
 
+  _createResourceRoutes() {
+    if (!this.options.resourceRoutes) return true;
+    const resourceRoutesOptions = this.options.resourceRoutes;
+    const resourceRoutes = parseResources(this.service.resources);
+
+    if (_.isEmpty(resourceRoutes)) return true;
+
+    this.printBlankLine();
+    this.serverlessLog('Routes defined in resources:');
+
+    Object.keys(resourceRoutes).forEach(methodId => {
+      let resourceRoutesObj = resourceRoutes[methodId],
+          path              = resourceRoutesObj.path,
+          method            = resourceRoutesObj.method,
+          isProxy           = resourceRoutesObj.isProxy,
+          proxyUri          = resourceRoutesObj.proxyUri,
+          pathResource      = resourceRoutesObj.pathResource;
+
+      if (!isProxy)
+        return this.serverlessLog(`WARNING: Only HTTP_PROXY is supported. Path '${pathResource}' is ignored.`);
+      if (`${method}`.toUpperCase() !== 'GET')
```

Comment:
Is there any reason to support only GET method? Can we support POST/DELETE or ANY?

Reply:
It's just because GET is the easiest one to implement.

To fully support POST, we need to worry about multipart requests.
 (https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4.2)